src/Bridge/Doctrine/Orm/Extension/OrderExtension.php

```diff
@@ -37,6 +37,7 @@ public function applyToCollection(QueryBuilder $queryBuilder, QueryNameGenerator
         $classMetaData = $queryBuilder->getEntityManager()->getClassMetadata($resourceClass);
         $identifiers = $classMetaData->getIdentifier();
 
+        //@TODO Why?
         if (null !== $this->order) {
             foreach ($identifiers as $identifier) {
                 $queryBuilder->addOrderBy('o.'.$identifier, $this->order);
```

Comment:
@dunglas Filter order will get on top of this but I was just wondering why is this necessary? Thanks